doc/starterkit/k4MarlinWrapperCLIC/Readme.md

```diff
@@ -124,14 +124,18 @@ included with k4MarlinWrapper. Note that:
   + Change line `evtsvc.input = '$TEST_DIR/inputFiles/ttbar1_edm4hep.root'` to point to the location of your input file.
   + At the bottom of the file, in the `ApplicationMgr` parameters, change `EvtMax   = 3,` to the number of events to run.
 
-
-This can be run with:
+This can be run in the following way. Note that we show the usage here in a way
+that also works in scripts with `-e` enabled. This is necessary because of a
+[framework issue](https://github.com/key4hep/k4FWCore/issues/125) that makes
+`k4run` exit with a non-zero exit code even though nothing went wrong.
 
 ```bash
 cd CLICPerformance/clicConfig
 
 cp ../../../test/gaudi_opts/clicRec_e4h_input.py .
 
 k4run clicRec_e4h_input.py --EventDataSvc.input ttbar_edm4hep.root
+result=$?
+[ $result = "0" ] || [ $result = "4" ] && true
```

Comment:
Same comment as above, not needed for most people so it can be separated from the code that is going to be copied and pasted most of the time